venom/rpc/reflect/stubs.py

```diff
@@ -1,10 +1,62 @@
 from venom import Message
 from venom.rpc import Stub
 from venom.rpc import http
+from venom.fields import Field, Repeat, String, Map, Bool
+
+
+class TypeMessage(Message):
+    type = String()  # TODO: enum
+    description = String()
+    items = Field('venom.rpc.reflect.openapi.TypeMessage')
+    ref = String(name='$ref')
+    additionalProperties = Field('venom.rpc.reflect.openapi.TypeMessage')
```

Comment:
Don't use camelCase, use snake_case. The JSON encoder ~automatically converts~ should automatically convert fields to camelCase.